Userland/Libraries/LibRegex/RegexMatcher.cpp

```diff
@@ -415,7 +415,6 @@ template<class Parser>
 bool Matcher<Parser>::execute(MatchInput const& input, MatchState& state, size_t& operations) const
 {
     BumpAllocatedLinkedList<MatchState> states_to_try_next;
-    size_t recursion_level = 0;
```

Comment:
You you missed the usage in under `#if REGEX_DEBUG` below.

Reply:
"Curse you, `ALL_DEBUG`!"

Should be fixed now.